test.js

```diff
@@ -61,3 +61,17 @@ test('npmKeyword.count()', async t => {
 	t.is(typeof cnt1, 'number');
 	t.is(cnt2, 0);
 });
+
+test('npmKeyword.count() using an array of keywords', async t => {
```

Comment:
Would love to test the part of the URL separately.
But that would mean to move some code to `lib/search-url.js, right?

Reply:
You mean you want to test https://github.com/mischah/npm-keyword/blob/ebad0e848901f4d5495e9d605ffbfafd3ae6f528/index.js#L16 separately? For what purpose?

Reply:
To make the testing the correct URL creation independent of the API and the response. 
I’m not sure if this is necessary 🤔 

What do you mean?